tests/listediting.js

```diff
@@ -3625,7 +3625,7 @@ describe( 'ListEditing', () => {
 			const uiElement = new ViewUIElement( 'span' );
 
 			// Append ui element at the end of first <li>.
-			viewDoc.getRoot().getChild( 0 ).getChild( 0 ).appendChildren( [ uiElement ] );
+			viewDoc.getRoot().getChild( 0 ).getChild( 0 )._appendChildren( [ uiElement ] );
```

Comment:
I think we should use `view.change` and `model.change` in tests too. It is now not a good practice I think and this might encourage others to use protected methods in tests. WDYT?